infrastructure/views/api.py

```diff
@@ -108,12 +107,13 @@ class ProjectSearch(generics.ListCreateAPIView):
     serializer_class = serializers.ProjectSerializer
     pagination_class = PageNumberPagination
     fieldmap = {
-        "municipality": "geography__name",
+        "geography__name": "municipality",
         "function": "function",
         "project_type": "project_type",
-        "province": "geography__province_name",
-        "budget_phase": "expenditure__budget_phase__name",
-        "financial_year": "expenditure__financial_year__budget_year",
+        "geography__province_name": "province",
+        "expenditure__budget_phase__name": "budget_phase",
+        "expenditure__financial_year__budget_year": "financial_year",
+        "latest_implementation_year__budget_year": "financial_year",
```

Comment:
are we supposed to have both of these? it looks like this means we'll do

```
qs.filter(
    expenditure__financial_year__budget_year=qsparam, 
    latest_implementation_year__budget_year=qsparam
)
```

which means we'll only get projects that both match the provided financial year in their latest_implemetation_year and also has an expenditure item for that year. we don't care about the latter though.

Reply:
```suggestion
        "latest_implementation_year__budget_year": "financial_year",
```

Reply:
I agree but I think this is outside the scope of this PR which is to be able to filter projects by a configurable latest implementation year. This should be fixed in #553 where annual and quarterly uploads should display projects that don't necessarily have a 'Budget year' expenditure for the latest implementation year